tableTheory/tableTheory.tex

```diff
@@ -309,17 +349,17 @@ \subsubsection{Bag Count}
 \begin{itemize}
 \item
 We do not include an explicit membership predicate for bags, although note that
-the formula $\syntax{(>\ (count\ u\ t)\ 0)}$ holds exactly when $\syntax{u}$
+the formula $\syntax{(>\ (bag.count\ u\ t)\ 0)}$ holds exactly when $\syntax{u}$
```

Comment:
cvc5 now reads `(bag.member u t)` as a syntax sugar for `(> (bag.count u t) 0)`

Reply:
We can add this and other syntax sugar later.